apps/vs-code-intellisense/src/extension.ts

```diff
@@ -2,19 +2,20 @@
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
 
-import pasteTokens, {pasteTokenAttributes} from './tokens';
-import {PasteToken} from './models/paste-token';
+import {pasteTokenAttributes} from './tokens';
+import {PasteToken} from './types';
+import {getThemeSetting, getThemeTokens, remToPx} from './utils';
+import kebabCase from 'lodash.kebabcase';
```

Comment:
I imported according to the rules:
1. import libraries `import * as vscode from 'vscode';`
2. import static paths
3. import relative paths `import {PasteToken} from './types';`
Could you keep the same imports?

Reply:
Should be fixed in each file now!